web/themes/custom/fmcg_material/sass/styles.scss

```diff
@@ -2192,3 +2308,36 @@ form#views-exposed-form-job-board-page {
   right: 0;
   top: 0;
 }
+
+.tabs {
+  background-color: $fmcg-black;
+  border-bottom: 1px solid $fmcg-yellow;
+}
+
+li.tab.is-active {
+  background-color: #FDD22B;
+}
+
+thead {
+  background-color: $fmcg-black;
+  border-bottom: 1px solid $fmcg-yellow;
+
+  th {
+    font-weight: 400;
+    color: $fmcg-chrome;
+    text-align: left;
+  }
+}
+
+.field-name-name {
```

Comment:
A bit concerned about this selector? What is it doing?

Reply:
This is the field that contains the organization name, that appears above the recruiter dashboard tabs.

Reply:
Can we wrap it in a class associated with the block then? Just don't want this to apply to every field called "name" anywhere in the website.

Reply:
@BenPoole88 this doesn't seem to have been done?